scylla/src/network/connection.rs

```diff
@@ -451,7 +480,7 @@ impl Connection {
         }
 
         // TODO: What should be the size of the channel?
-        let (sender, receiver) = mpsc::channel(1024);
+        let (sender, receiver) = mpsc::channel(config.pending_request_channel_size.unwrap_or(1024));
         let (error_sender, error_receiver) = tokio::sync::oneshot::channel();
```

Comment:
Please remove the TODO comment. 
Also you are doing `unwrap_or(1024)`, but you wrote in PR description that you changes the default to 2048 - it looks like a mistake.